Lesson8/main.js

```diff
@@ -0,0 +1,22 @@
+const addLoading = () =>{
+  const loading = document.getElementById("js-loading");
+  const loadingImg = document.createElement('img');
+  loadingImg.src = 'loading.gif';
+  loadingImg.id = "loading-img"
+  loading.appendChild(loadingImg)
+}
+
+const promiseChecker = new Promise((resolve, reject) => {
```

Comment:
Q
How about the same name as lesson7?
```suggestion
const promiseFeatures = new Promise((resolve, reject) => {
```

Reply:
Thank you for your suggestion Mai-san,
I changed name back to same as the lesson7. 
I am still learning what is the best practice to name variable, so your feeback is vert appreciated. 
Please check my latest commit 183faeaf363c02096a4749af382e02758a5fe22f